ethcore/src/block.rs

```diff
@@ -398,26 +392,11 @@ impl<'x> OpenBlock<'x> {
 
 	/// Turn this into a `ClosedBlock`.
 	pub fn close(self) -> Result<ClosedBlock, Error> {
-		let mut s = self;
-
-		let unclosed_state = s.block.state.clone();
-
-		s.engine.on_close_block(&mut s.block)?;
```

Comment:
Where does this callback happen after this change?

Reply:
https://github.com/paritytech/parity-ethereum/pull/9252/files#diff-e2e171033e005c3b329818aaff213767R408